cs/ccxt/base/Exchange.Generic.cs

```diff
@@ -72,7 +74,7 @@ public List<object> filterBy(object aa, object key, object value)
         var outList = new List<object>();
         foreach (object elem in targetA)
         {
-            if (((dict)elem)[(string)key].ToString() == value.ToString())
+            if (((dict)elem)[(string)key]?.ToString() == value?.ToString())
```

Comment:
in other langs, we don't have `.ToString()` as those values can be null, so in `c#` it was throwing errors, so we should do this way.